examples/config-dynamodb.json

```diff
@@ -19,7 +19,8 @@
             "region": "us-east-2",
             "cacheTablename": "ZotBlobTable",
             "repoMetaTablename": "ZotRepoMetadataTable",
-            "manifestDataTablename": "ZotManifestDataTable",
+            "imageDataTablename": "ZotImageDataTable",
```

Comment:
"imageMeta.."

Reply:
Line 22 lookup(digest) -> record

Reply:
I thought about this naming issue and it would be very useful to name this table "ImageMetadata" so I think we should keep it as it is. I am using the term "ImageMetadata" as something else with more information than just the "data" we gather form the layout